lib/matplotlib/axes/_base.py

```diff
@@ -3397,6 +3397,13 @@ def get_xticklabels(self, minor=False, which=None):
         -------
         ret : list
            List of `~matplotlib.text.Text` instances.
+        
+        Notes
+        -----
+        Please note that the tick label strings are not populated until a ``draw``
+        method has been called.
+
+        See also: `~pyplot.draw` and `~canvas.draw`.
```

Comment:
```suggestion
        See also: `~.pyplot.draw` and `~.FigureCanvasBase.draw`.
```

Reply:
Thanks, I'm still learning ;-) I wasn't sure how to get it to do cross-references exactly and must have overlooked the period copying from another example...